class/Utils/Utils.php

```diff
@@ -4,6 +4,7 @@
 
 class Utils
 {
+  private const PASSLE_SYNC_CACHE_KEY = "passlesync_posts_cache";
```

Comment:
I am not sure this is the right way to recover the posts cache key - this is defined elsewhere and having it hardcoded here will probably break if one of the instances of this changes and re-introduce this bug.